frontend/src/components/views/SideBar/TSideBar.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <t-animation>
+    <aside class="side" v-show="isSidebarVisible">
+      <t-side-bar-cluster-dropdown
+        v-click-outside="onClickOutside"
+        v-show="isSidebarClusterDropdownsActive"
+        :contexts="contexts"
+      />
+      <div
+        ref="dropdown"
+        class="side__cluster-box"
+        @click="
+          () =>
+            isSidebarClusterDropdownsActive
+              ? (isSidebarClusterDropdownsActive = false)
+              : (isSidebarClusterDropdownsActive = true)
+        "
+      >
+        <div ref="dropdownWrapper" class="side__wrapper">
+          <p class="side__claster-name"><slot /></p>
+          <p class="side__description">Cluster API</p>
+        </div>
+        <t-icon class="side__arrow" icon="arrow-right" />
+      </div>
+      <t-side-bar-list />
+      <div class="side__disclosure">
+        <component :is="sidebar" v-if="sidebar" />
+      </div>
+    </aside>
+  </t-animation>
+</template>
+
+<script lang="ts">
+import { computed, ref, watch } from "@vue/runtime-core";
+import { useRoute } from "vue-router";
+import TIcon from "../../../components/common/Icon/TIcon.vue";
+import TAnimation from "../../common/Animation/TAnimation.vue";
+import TSideBarClusterDropdown from "../SideBar/components/TSideBarClusterDropdown.vue";
+import TSideBarList from "./components/TSideBarList.vue";
+import vClickOutside from "click-outside-vue3";
+import TSideBarNodesLogs from "./components/TSideBarNodesLogs.vue";
+import { getTSideBarNodesLogs } from "../../../router";
+
+export default {
+  directives: {
+    clickOutside: vClickOutside.directive,
+  },
+  components: {
+    TIcon,
+    TAnimation,
+    TSideBarClusterDropdown,
+    TSideBarList,
+    TSideBarNodesLogs,
+  },
+  props: {
+    contexts: Array,
+  },
+  setup() {
+    let isSidebarVisible = ref(true);
+    const isSidebarClusterDropdownsActive = ref(false);
+    const dropdown = ref(null);
+    const dropdownWrapper = ref(null);
+    const route = useRoute();
+    const sidebar = ref<any>(getTSideBarNodesLogs(route));
+
+    isSidebarVisible = computed(() => {
+      return route.path !== "/dashboard";
+    });
+
+    watch(
+      () => route.params,
+      () => {
+        sidebar.value = getTSideBarNodesLogs(route);
+      }
+    );
+
+    watch(
+      () => route.query,
+      () => {
+        sidebar.value = getTSideBarNodesLogs(route);
+      }
+    );
```

Comment:
I wonder if the following works the same:

```suggestion
    watch(
      [route.params, route.query],
      () => {
        sidebar.value = getTSideBarNodesLogs(route);
      }
    );
```